modules/govuk_jenkins/manifests/jobs/bouncer_cdn.pp

```diff
@@ -18,17 +18,19 @@
   $service_id = undef,
   $app_domain = hiera('app_domain'),
 ) {
-
-  $check_name = 'bouncer-cdn-configuration'
   $service_description = 'Configure Bouncer CDN service with transitioning sites'
-  $job_url = "https://deploy.${app_domain}/job/Bouncer_CDN/"
 
   file { '/etc/jenkins_jobs/jobs/bouncer_cdn.yaml':
     ensure  => present,
     content => template('govuk_jenkins/jobs/bouncer_cdn.yaml.erb'),
     notify  => Exec['jenkins_jobs_update'],
   }
 
+  $check_name = 'bouncer-cdn-configuration'
+
+  # temporary solution while we have two Icinga instances
```

Comment:
I'm not sure setting the `job_url` per environment is a temporary thing, or how it relates to Icinga?

Reply:
It's what controls the cog icon in Icinga, and ideally we could switch back to `app_domain` when we only have one Icinga instance AWS... but I can look at improving the comment.